contrib/lua/premake5.lua

```diff
@@ -21,8 +21,8 @@ project "lua-lib"
 		"etc/*.c"
 	}
 
-	configuration "linux or bsd or hurd or aix"
+	filter "system:linux or system:bsd or system:hurd or system:aix"
```

Comment:
Since filters are seperated by " or " it only applies to the first term, maybe being able to use ( and ) inside could help categorize things in a better way (same as it does in actual lua code)